CruCentralCoast/Community/Missions/MissionDetailsVC.swift

```diff
@@ -16,6 +17,8 @@ class MissionDetailsVC: UIViewController {
     @IBOutlet weak var locationLabel: UILabel!
     @IBOutlet weak var summaryLabel: UILabel!
     
+    var missionURLString: String?
```

Comment:
Since other classes don't need to access this variable, it should be made private

Reply:
good point thanks!